src/persist-types/src/stats.rs

```diff
@@ -546,6 +551,8 @@ impl TrimStats for ProtoStructStats {
                 Some(Kind::Primitive(stats)) => stats.trim(),
                 Some(Kind::Bytes(stats)) => stats.trim(),
                 Some(Kind::Struct(stats)) => stats.trim(),
+                // Nothing to trim for NoneStats.
+                Some(Kind::None(_)) => (),
```

Comment:
nit: If you do the `google.protobuf.Empty` comment, pretty sure that codegens to `()`, which could be inlined here in the match (`Some(Kind::None(())) => (),`). And then we probably don't need the comment

Reply:
Yeah good call, updated!